atom/browser/browser.h

```diff
@@ -167,6 +167,13 @@ class Browser : public WindowListObserver {
   // one from app's name.
   // The returned string managed by Browser, and should not be modified.
   PCWSTR GetAppUserModelID();
+
+  typedef HRESULT (STDAPICALLTYPE *DwmGetColorizationColor)(DWORD *, BOOL *);
```

Comment:
Basically a whole bunch of unresolved identifiers, it didn't know what `DwmGetColorizationColor` was.  I'll try harder to work around it but adding one `typedef` didn't seem to bad at the time 👍